packages/astro/src/remark/import-file.ts

```diff
@@ -48,6 +48,7 @@ export function remarkImportFilePlugin(options: RemarkImportFilePluginOptions) {
 
           if (content) {
             node.value = content;
+            node.meta += ' path=' + node.lang;
```

Comment:
I'm not that familiar with expressive code's `node.meta`, but looking at the typings this can be `null | undefined`. We should probably do

```diff
+node.meta ||= '';
node.meta += ' path=' + node.lang;
```

... to avoid getting `'undefined path=somelang'`.

Reply:
good call. updated